src/EPR.RegulatorService.Facade.Core/Models/Accounts/EmailModels/OrganisationRegistrationSubmissionEmailModel.cs

```diff
@@ -33,10 +33,21 @@ public Dictionary<string, object> GetParameters {
                     { "organisation_number", this.OrganisationNumber },
                     { "organisation_name", this.OrganisationName },
                     { "agency", this.Agency },
-                    { "agency_email", this.AgencyEmail },
                     { "year", this.Period }
                 };
 
+                if (IsWelsh.Value)
+                {
+                    var emails = this.AgencyEmail.Split(';');
```

Comment:
better to have a null check before picking the items from array?

Reply:
Value will only return the value if it has been assigned to a nullable type.  So no need for null check.  Unless I am mistaken